services/showcase/api/db/migrations/versions/0e7a0e8f75ad_add_issuers_bools.py

```diff
@@ -0,0 +1,80 @@
+"""add issuers bools
+
+Revision ID: 0e7a0e8f75ad
+Revises: 3449c639cdbd
+Create Date: 2022-02-25 19:22:56.487174
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "0e7a0e8f75ad"
+down_revision = "3449c639cdbd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
+        "outofband", "sender_id", existing_type=postgresql.UUID(), nullable=True
+    )
+    op.alter_column(
+        "outofband", "recipient_id", existing_type=postgresql.UUID(), nullable=True
+    )
+    op.alter_column(
+        "outofband", "sandbox_id", existing_type=postgresql.UUID(), nullable=True
```

Comment:
I did not touch these columns, any idea why alembic wants to change from `nullable=false` to `nullable=true`?

Reply:
It is because you are generating based on the model class. The model class has those ids as optional because they are not part of the create schema (sandbox_id) or are to be optional on other schemas (update). you have to be very careful when generating migrations. (I don't bother, just code it by hand)